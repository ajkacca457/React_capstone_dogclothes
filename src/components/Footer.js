import React from 'react';

const Footer = () => (
  <div className="footer text-white pt-3">
    <p>&copy; & Project made by Avijit, Microverse</p>
  </div>
);

export default Footer;
