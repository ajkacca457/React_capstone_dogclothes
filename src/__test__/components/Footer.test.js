import React from 'react';
import TestRenderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Footer from '../../components/Footer';

describe('Footer Component', () => {
  it('renders correctly', () => {
    const quotetree = TestRenderer
      .create(<Footer />)
      .toJSON();
    expect(quotetree).toMatchSnapshot();
  });

  test('renders the title text of Footer Component', () => {
    const { getByText } = render(<Footer />);
    const title = getByText('Â© & Project made by Avijit, Microverse');
    expect(title).toBeInTheDocument();
  });
});
