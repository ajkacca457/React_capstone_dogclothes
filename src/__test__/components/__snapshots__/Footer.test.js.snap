// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component renders correctly 1`] = `
<div
  className="footer text-white pt-3"
>
  <p>
    Â© & Project made by Avijit, Microverse
  </p>
</div>
`;
